src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,1010 @@
+# Microelectrode Electrophysiology
+
+Support for Microelectrode Electrophysiology was developed as a [BIDS Extension Proposal](../extensions.md#bids-extension-proposals) [BEP032: Animal electrophysiology (ephys)](https://bids.neuroimaging.io/bep032).
+Please see [Citing BIDS](../introduction.md#citing-bids) on how to appropriately credit this extension
+when referring to it in the context of the academic literature.
+
+This BEP was initiated by members of the INCF Working Group on Standardized Data Structures in 2020
+to develop specifications and tools for standardizing experimental data recorded with animal models
+in neuroscience and its associated metadata.
+
+!!! example "Example datasets"
+
+    Several [example microelectrode electrophysiology datasets](https://bids-standard.github.io/bids-examples/#microephys)
+    have been formatted using this specification
+    and can be used for practical guidance when curating a new dataset.
+
+## Terminology: Modality and Datatypes
+
+The Microelectrode Electrophysiology modality encompasses recordings made with micrometer-scale electrodes,
+distinguishing it from related BIDS modalities (EEG, MEG, iEEG) that use larger electrodes.
+This modality is primarily used in animal research.
+
+Within this modality, BIDS defines two datatypes based on fundamentally different recording techniques (see [Issue #1800](https://github.com/bids-standard/bids-specification/issues/1800)):
+
+-   **`ecephys`** (Extracellular Electrophysiology): Electrodes remain in the extracellular space,
+    measuring field potentials (μV) from nearby neurons without membrane penetration.
+    Examples: Neuropixels probes, tetrodes, multi-electrode arrays.
+
+-   **`icephys`** (Intracellular Electrophysiology): Electrodes penetrate or attach to cell membranes to directly measure
+    intracellular potentials (mV) and cellular dynamics.
+    Examples: cell-attached patch clamp, whole-cell patch clamp, sharp electrode recordings.
```

Comment:
sharp electrodes can also be used to record extracellular, especially when the exposed tip is large enough (1 um or larger). See for example https://www.thomasrecording.com/np-electrodes-fm-single-electrodes.

I find this "sharp electrode recordings" example therefore more confusing than clarifying, and therefore suggest to drop it.